low-level/files/opt/seagate/sspl/bin/sspl_post_install

```diff
@@ -115,5 +115,5 @@ cp -R $SSPL_BASE_DIR/resources/iem/iec_mapping/* $PRODUCT_BASE_DIR/iem/iec_mappi
 # already configured on the healthy node
 # Configure rabbitmq
 [ -f $REPLACEMENT_NODE_ENV_VAR_FILE ] || {
-    [ "$RMQ_CLUSTER" == true ] && $script_dir/rabbitmq_setup
+    [ "$RMQ_CLUSTER" != true ] || $script_dir/rabbitmq_setup
```

Comment:
why this change? Its accomplishing same anyways !

Reply:
I would suggest to take this up in another patch if needed. let us add only sspl_update file in this patch.